src/Catalog/LogEvents.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public static class LogEvents
+    {
+        public static EventId ValidationFailed = new EventId(900, "Validation failed!");
```

Comment:
There's no logger extensions for `LogError`/`LogCritical` with exception without an `EventId`. We have this pattern in `NuGet.Jobs` in a couple places.